puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -0,0 +1,61 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+                checkout changelog: true, poll: false, scm: [
```

Comment:
I think we can generalize this now or as a follow up item as its used in 3-4 places now we've got a pattern established

Reply:
done